frontend/test/metabase/scenarios/models/model-actions.cy.spec.js

```diff
@@ -154,6 +154,34 @@ describe("scenarios > models > actions", () => {
     );
   });
 
+  it("should allow to create an action with the New button", () => {
+    const QUERY = "UPDATE orders SET discount = {{ discount }}";
+    cy.visit("/");
+
+    cy.findByText("New").click();
+    popover().findByText("Action").click();
+
+    cy.findByText("Select a database").click();
+    popover()
+      .findByText(/QA Postgres/i)
```

Comment:
nit: we should use a simple string unless it's a partial match. From a quick search I think the full db name is `"QA Postgres12"`